modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/HttpApi.scala

```diff
@@ -121,6 +121,8 @@ sealed abstract class HttpApi[F[_]: Async: SecurityProvider: KryoSerializer: Has
   private val metricRoutes = MetricRoutes[F]().publicRoutes
   private val targetRoutes = TargetRoutes[F](services.cluster).publicRoutes
 
+  private val metagraphMessageRoutes = new MetagraphMessageRoutes[F](services.metagraphMessageService).publicRoutes
```

Comment:
afaik we use `data application` name within the codebase, not `metagraph` 🤔 is the functionality clearly for metagraphs and you distinguish the metagraph from any other data application?

Reply:
renamed to use prefix "Currency", hopefully changed all references.